src/pudl/metadata/resources/eiaaeo.py

```diff
@@ -0,0 +1,524 @@
+                "nominal_price_per_mwh",
+                "real_price_per_mwh",
+            ],
+            "primary_key": [
+                "report_year",
+                "electricity_market_module_region_aeo",
+                "modeling_case_aeo",
+                "projection_year",
+                "customer_class",
+            ],
+        },
+        "sources": ["eiaaeo"],
+        "field_namespace": "eiaaeo",
+        "etl_group": "eiaaeo",
+    },
+    "core_eiaaeo__yearly_projection_end_use_prices_by_service_category": {
+        "description": (
+            "Projected electricity cost to the end user by service category."
+        ),
+        "schema": {
+            "fields": [
+                "report_year",
+                "electricity_market_module_region_aeo",
+                "modeling_case_aeo",
+                "projection_year",
+                "service_category_aeo",
+                "nominal_price_per_mwh",
+                "real_price_per_mwh",
+            ],
+            "primary_key": [
+                "report_year",
+                "electricity_market_module_region_aeo",
+                "modeling_case_aeo",
+                "projection_year",
+                "service_category_aeo",
+            ],
+        },
+        "sources": ["eiaaeo"],
+        "field_namespace": "eiaaeo",
+        "etl_group": "eiaaeo",
+    },
+    "core_eiaaeo__yearly_projection_fuel_consumption_by_type": {
+        "description": (
+            "Projected fuel consumption by the electric power sector, "
+            "including electricity-only and combined-heat-and-power plants "
+            "that have a regulatory status."
+        ),
+        "schema": {
+            "fields": [
+                "report_year",
+                "electricity_market_module_region_aeo",
+                "modeling_case_aeo",
+                "projection_year",
+                "fuel_type_aeo",
+                "fuel_consumed_mmbtu",
+            ],
+            "primary_key": [
+                "report_year",
+                "electricity_market_module_region_aeo",
+                "modeling_case_aeo",
+                "projection_year",
+                "fuel_type_aeo",
+            ],
+        },
+        "sources": ["eiaaeo"],
+        "field_namespace": "eiaaeo",
+        "etl_group": "eiaaeo",
+    },
+    "core_eiaaeo__yearly_projection_electric_sector_fuel_cost_by_type": {
+        "description": (
+            "Projected fuel prices to the electric power sector, including "
+            "electricity-only and combined-heat-and-power plants that have a "
+            "regulatory status."
+        ),
+        "schema": {
+            "fields": [
+                "report_year",
+                "electricity_market_module_region_aeo",
+                "modeling_case_aeo",
+                "projection_year",
+                "fuel_type_aeo",
+                "fuel_cost_per_mmbtu",
+                "fuel_cost_real_per_mmbtu_aeo",
```

Comment:
did you add the source suffix here bc the real-ness is source dependent? that seems realonable ;-)

Reply:
Yeah, it just seemed funny to have "real" as its own field divorced from context - if it's AEO at least we know that the associated year is `report_year-1`. Outside of an AEO context this just doesn't really mean anything...